### DISCUSSION TITLE
Add new handler to get if script environments is creating

### DISCUSSION THREAD

**Discussion by TheD4mn** [citizenfx/fivem#1807] [Engineering Discussion]
I think it would be usefull to get if script environments is creating, because not all the scripts can be getted from onResourceStart or other handlers, so it would be usefull to have handler like 'onCreateScriptEnvironments' or something like that.

**Comment by mickeygeecom** [comment]
[onResourceStarting](https://docs.fivem.net/docs/scripting-reference/events/server-events/#onresourcestarting)

This might be what you are looking for
